doc/guides/collaborator-guide.md

```diff
@@ -143,30 +143,28 @@ the TSC meeting agenda.
 
 ### Waiting for Approvals
 
-Before landing pull requests, allow 48 hours for input from other Collaborators.
-Certain types of pull requests can be fast-tracked and may land after a shorter
-delay. For example:
+Before landing pull requests, allow for input from other Collaborators.
+If you expect that a pull request might raise objections, it is a good idea to
+wait longer before landing. You can also add the `wait-for-feedback` label
+to pull requests; in that case, it is expected that the pull request will not
+be merged until at least 48 hours after it has been opened, in order to allow
+for everybody to comment on it.
 
-* Focused changes that affect only documentation and/or the test suite:
-  * `code-and-learn` tasks often fall into this category.
-  * `good-first-issue` pull requests may also be suitable.
-* Changes that fix regressions:
-  * Regressions that break the workflow (red CI or broken compilation).
-  * Regressions that happen right before a release, or reported soon after.
+For pull requests that are lacking an approval from a member of the relevant
+team (e.g. @nodejs/http in the case of HTTP changes), it is recommended to keep
```

Comment:
Is this easily visible in the PR if such a member has approved?

Reply:
No, unfortunately not. I think it would help if the teams are being pinged, though, since that allows you to see the list of members more easily.

Reply:
```suggestion
team (e.g. @nodejs/http in the case of HTTP changes), keep
```

Reply:
This is not something easy. The contributor should learn on whom to ping. For example, ping this person for util related PR. It would definitely take lot of time. That's how I have came to know the codeowners for few such modules.  

May be can we mention such codeowners in an readme file or something like that? I guess the new contributors can't ping our groups directly until they are part of our organization.

Reply:
The bot pings codeowners now, so this issue should be mitigated.